src/Toc.ts

```diff
@@ -26,7 +26,14 @@ export default class Toc {
     this.spec.doc.body.setAttribute('class', bodyClass + ' oldtoc');
   }
 
-  static build(level: Spec | Clause, expandy?: boolean) {
+  static build(level: Spec | Clause, options: { maxDepth?: number, expandy?: boolean } = {}) {
```

Comment:
If you destructure in place you can set the defaults inline and skip the explicit type:
```suggestion
  static build(level: Spec | Clause, { maxDepth = Infinity, expandy = false } = {}) {
```